acceptance/features/migration.feature

```diff
@@ -0,0 +1,32 @@
+Feature: operator can migrate/adopt Redpanda helm base deployment
+
+  @skip:gke @skip:aks @skip:eks
+  Scenario: Adopt or migrate from stand alone helm chart release to Redpanda custom resource
+        Given a Helm release named "redpanda-migration-example" of local Redpanda Helm Chart with values:
```

Comment:
Given a local Helm release named "redpanda-migration-example"  with the following values: